src/utils/index.js

```diff
@@ -6,18 +6,25 @@ const SKINS = [
   '1F3FD', '1F3FE', '1F3FF',
 ]
 
-function unifiedToNative(unified) {
+function unifiedToNative(unified,custom) {
```

Comment:
i dont exactly understand that unified is.. changed `unified  = name` for custom emojis... pre-loaded emojis are unnaffected by this.